e2e/test_elfdeps.sh

```diff
@@ -14,7 +14,7 @@ VERSION="2.1.5"
 OS=$(uname)
 if [ "$OS" = "Darwin" ]; then
     NETWORK_ISOLATION=""
-    WHEEL_PLATFORM_TAG="macosx_10_9_universal2"
+    WHEEL_PLATFORM_TAG="macosx_10_13_universal2"
```

Comment:
I have a different fix for this in #420 that should be future proof and allow the tests to work on my dev box, too.

Reply:
I have merged main into the branch and resolved the merge conflict.